src/components/OrgSettings/ActionItemCategories/OrgActionItemCategories.tsx

```
@@ -256,36 +292,47 @@ const OrgActionItemCategories: FC<InterfaceActionItemCategoryProps> = ({
-                'edit',
-              )
-            }
-          >
-            <i className="fa fa-edit" />
-          </Button>
+          <div className="d-flex gap-2 justify-content-center align-items-center h-100">
+            <Button
+              variant="success"
+              size="sm"
+              className={styles.editButton}
+              data-testid={'viewCategoryBtn' + params.row.serialNumber}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)